src/constants.ts

```diff
@@ -29,5 +30,12 @@ export enum configurationSettings {
     zipIgnorePattern = 'zipIgnorePattern',
     showBuildDuringDeployPrompt = 'showBuildDuringDeployPrompt',
     deploySubpath = 'deploySubpath',
-    advancedCreation = 'advancedCreation'
+    advancedCreation = 'advancedCreation',
+    deployToWebAppId = 'deployToWebAppId'
+}
+
+export enum deploymentEntryPoint {
```

Comment:
So we don't want to just throw EVERY constant into this file. I would only put it in here if it meets either of the following:
1. It's used multiple times
1. It's only used once, but we expect to change it (i.e. maybe a version number) and we want it to be easy to find

I don't think this meets either of those qualifications